src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Source.cs

```diff
@@ -54,7 +54,7 @@ private static void FreeSymbolMap(MultiDictionary<string, ISymbol> symbolMap)
 
             // Order the documents by FilePath.  Default ordering in the RemoteWorkspace is
```

Comment:
@sharwell I think yes: if this isn't preserving document order, that means the out-of-proc compilations will have differing behaviors than in-proc, and that could be bad. #20677 filed to investigate further.